modules/installation-special-config-raid.adoc

```diff
@@ -19,7 +32,11 @@ version: 4.10.0
 metadata:
   name: raid1-storage
   labels:
-    machineconfiguration.openshift.io/role: worker
+    machineconfiguration.openshift.io/role: raid-worker
+ignition:
+  config:
+    merge:
+      - source: http://cluster-name:22623/worker.ign
```

Comment:
I'm not an SME on the MCS.  Is the worker config always hosted at this URL?

Reply:
yes: https://github.com/openshift/machine-config-operator/blob/5516bd8911ce3e53cfc20924609c129d2eabc604/pkg/server/api.go#L23

Reply:
it seems to always be located at that port, though it looks like it lives in a [`config`](https://github.com/openshift/machine-config-operator/blob/2fb7a6bae2b51c79f77fd939b693a64f8608b8e2/docs/HACKING.md#accessing-the-machineconfigserver-directly) directory. I'll fix that